go.mod

```diff
@@ -8,6 +8,7 @@ require (
 	github.com/prometheus/client_golang v1.20.5
 	github.com/segmentio/kafka-go v0.4.47
 	github.com/sirupsen/logrus v1.9.3
+	github.com/stretchr/testify v1.9.0
```

Comment:
We don't use this dependency, can you look the other tests which use t.Fatal etc.